packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1076,6 +1121,11 @@ export default function buildKernel(
     if (gcMessage) {
       return gcMessage;
     }
+    const boydMessage = kernelKeeper.nextBoydAction();
```

Comment:
Good, by pulling `nextBoydAction` *after* `processNextGCAction`, we'll deliver any `dispatch.dropExports` before we get around to reaping, which should maximize the amount of garbage than can be collected.